### ISSUE TITLE
Nuxt svg logos are buggy

### ISSUE THREAD

**Issue by dennis47528** [nuxt/nuxtjs.org#274]
### Version

[v1.1.0](https://github.com/nuxtjs.org/releases/tag/v1.1.0)

### Reproduction link

[https://nuxtjs.org/design](https://nuxtjs.org/design)

### Steps to reproduce

Look at the Nuxt logos in the following files:

- nuxt.svg
- nuxt-square.svg
- nuxt-white.svg
- nuxt-square-white.svg
- built-with-nuxt.svg
- built-with-nuxt-white.svg

### What is expected ?

The bugs shown in the 'What is actually happening? ' section below are fixed.

(🤔 This svg file doesn't have these weird corners: https://github.com/nuxt/nuxtjs.org/blob/master/nuxt.svg)

#### ⚠ nuxt.svg, nuxt-square.svg, nuxt-white.svg, nuxt-square-white.svg

![Anmerkung 2019-10-31 162905.png](https://cmty.s3.eu-west-1.amazonaws.com/73294699/636b9390-fbf3-11e9-9dff-dfe3a12c4d7d.png)

![Anmerkung 2019-10-31 162949.png](https://cmty.s3.eu-west-1.amazonaws.com/73294699/6cbe8240-fbf3-11e9-9dff-dfe3a12c4d7d.png)

![Anmerkung 2019-10-31 162531.png](https://cmty.s3.eu-west-1.amazonaws.com/73294699/b93e3530-fbf2-11e9-9dff-dfe3a12c4d7d.png)

#### ⚠ built-with-nuxt.svg, built-with-nuxt-white.svg

![Anmerkung 2019-10-31 160640.png](https://cmty.s3.eu-west-1.amazonaws.com/73294699/4aa968d0-fbf0-11e9-9dff-dfe3a12c4d7d.png)

![Anmerkung 2019-10-31 165544](https://user-images.githubusercontent.com/33426785/67963774-7209fc00-fbff-11e9-9d6e-a913887e97aa.png)

### Additional comments?

This needs to be fixed as soon as possible because people are downloading and using a buggy version of the logo. 
It looks like the png files are ok.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxtjs.org/issues/c184">#c184</a>)</em></sub></div>

**Comment by Atinux** [comment]
Dammit, thank you, will fix ASAP

**Comment by dennis47528** [comment]
Thank you 👍 @Atinux 
So many people used this buggy logo in their project, website etc. it's everywhere. Maybe you should make a small announcement on twitter with the nuxt account, so that everybody knows about this issue. Very sad that nobody noticed this ... for months 🙁

**Comment by ngblaylock** [comment]
I just noticed this today, and have created a file that could be ready to use. Just a few notes on it:

1. I assumed that the logo was going for equilateral triangles, which my file uses. The transparent image is the SVG from the website, and the red lines are guides for how the new SVG file line up together.
![nuxt](https://user-images.githubusercontent.com/16627408/68924193-72e17880-073d-11ea-90ad-6f4a12963adf.png)

2. The rounded corners are perfectly rounded (thanks to the software) but do not line up exactly with Nuxt's logo, so that might take more tweaking if it matters.

3. The logos have some unnecessary lines that aren't visually noticeable, but are easily cleaned up. This new SVG file has perfectly lined up paths and no extra nodes.
<img width="890" alt="Screen Shot 2019-11-15 at 12 06 43 AM" src="https://user-images.githubusercontent.com/16627408/68923643-01ed9100-073c-11ea-97c5-3013cf3006d1.png">

And here is the SVG that I have put together. Everything is as clean, workable, and scalable as a SVG was intended for.
[nuxt-logo.svg](https://nuxt.nathanblaylock.com/nuxt-logo.svg)
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 404.19 300.03"><defs><style>.cls-1{fill:#00c58e;}.cls-2{fill:#2f495e;}.cls-3{fill:#108775;}</style></defs><title>nuxt-logo</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M230.77,82.1l-13.86,24L169.51,24,24,276h98.52a24,24,0,0,0,24,24H24A24,24,0,0,1,3.22,264L148.73,12a24,24,0,0,1,41.57,0Z"/><path class="cls-2" d="M335.81,264,244.63,106.1l-13.86-24-13.86,24L125.74,264a24,24,0,0,0,20.78,36H315a24,24,0,0,0,20.79-36ZM146.52,276,230.77,130.1,315,276Z"/><path class="cls-3" d="M401,288a24,24,0,0,1-20.78,12H315a24,24,0,0,0,24-24h41.17L263.35,73.67,244.63,106.1l-13.86-24,11.8-20.43a24,24,0,0,1,41.57,0L401,264A24,24,0,0,1,401,288Z"/></g></g></svg>
```

I just started with Nuxt about two weeks ago and love it, so I am happy to help out however I can.
